src/views/StandaloneOfficeAddressFiling.vue

```diff
@@ -383,13 +380,33 @@ export default {
             if (filing.business.identifier !== this.entityIncNo) throw new Error('Invalid business identifier')
             if (filing.business.legalName !== this.entityName) throw new Error('Invalid business legal name')
 
-            // load Certified By but not Date
+            // load Certified By (but not Date)
             this.certifiedBy = filing.header.certifiedBy
 
             // load Staff Payment properties
-            this.routingSlipNumber = filing.header.routingSlipNumber
-            this.isPriority = filing.header.priority
-            this.isWaiveFees = filing.header.waiveFees
+            if (filing.header.routingSlipNumber) {
+              this.staffPaymentData = {
+                option: StaffPaymentOptions.FAS,
+                routingSlipNumber: filing.header.routingSlipNumber,
+                isPriority: filing.header.priority
```

Comment:
Even though this is not a problem functionally, code wise this chunk is repeated thrice. May be using store is a better option here

Reply:
Great point! Will look into this 👍

Reply:
Leaving this comment un-resolved for the meantime, as potentially there is some refactoring here. The work chunk looked significant enough that i felt it might be best done in the hands of the author. There is no functional impact here, but there might be a better option for code maintenance in the future.